File: 1011.py
def alphaCentauri(distance) : 
    i = 1
    result = 0
    while(True) :
        distance = distance - i
        if(distance <= 0) :
            result = 2 * i - 1
            break
        distance = distance - i
        if(distance <= 0) :
            result = 2 * i
            break
        i = i + 1
    return result

numOfInput = int(input())

for i in range(numOfInput) :
    departure, destination = map(int, input().split())
    distance = destination - departure;
    print(alphaCentauri(distance))
    

File: 1011.c
#include<stdio.h>

int alphaCentauri(int distance);

int main(){
    int numOfInput, departure, destination;
    scanf("%d", &numOfInput);
    for(int i = 0; i < numOfInput; i++){
        scanf("%d %d", &departure, &destination);
        printf("%d\n", alphaCentauri(destination - departure));
    }
    return 0;
}

int alphaCentauri(int distance){
    int i = 1;
    int result;
    while(1){
        distance = distance - i;            //I JUMPS
        if(distance <= 0){                  //IF IN RANGE OF I JUMPS
            result = 2 * i - 1;             
            break;
        }
        distance = distance - i;            //ADDITIONAL I JUMPS
        if(distance <= 0){                  //IN RANGE OF ADDITIONAL I JUMPS
            result = 2 * i;
            break;
        }
        i++;
    }
    return result;
}
